### ISSUE TITLE
Is this model image only or can it be used with text conversation as well?

### ISSUE THREAD

**Issue by git4sun** [THUDM/VisualGLM-6B#31]
As the title, is this model only support conversations with images? If I want to just chat, do I need to run the chatglm model separately? The code seems require `image` field in the input. Setting it to empty or ignore will generate error.

**Comment by Sleepychord** [comment]
Yes, the model can support chat without images. For example in `cli_demo.py`, you can type "Enter" to skipping the image path input. But in web/api demos, you might need to manually change the input logic.
